docs/hub/miner-index.md

```diff
@@ -0,0 +1,676 @@
+              "mibPerSec": 20.966012990699625
+            },
+            {
+              "transferedAt": "2021-02-23T02:00:08Z",
+              "mibPerSec": 12.749545486246953
+            }
+          ],
+          "tailSealed": [
+            {
+              "sealedAt": "2021-03-17T14:56:51Z",
+              "durationSeconds": "9440"
+            },
+            {
+              "sealedAt": "2021-03-16T14:02:38Z",
+              "durationSeconds": "7200"
+            },
+            {
+              "sealedAt": "2021-03-16T13:56:09Z",
+              "durationSeconds": "7200"
+            },
+            {
+              "sealedAt": "2021-03-06T05:49:39Z",
+              "durationSeconds": "9000"
+            },
+            {
+              "sealedAt": "2021-03-06T05:49:37Z",
+              "durationSeconds": "9000"
+            },
+            {
+              "sealedAt": "2021-03-06T05:46:35Z",
+              "durationSeconds": "9000"
+            },
+            {
+              "sealedAt": "2021-03-06T05:45:37Z",
+              "durationSeconds": "9000"
+            },
+            {
+              "sealedAt": "2021-03-04T07:00:50Z",
+              "durationSeconds": "9000"
+            },
+            {
+              "sealedAt": "2021-03-03T22:40:39Z",
+              "durationSeconds": "9000"
+            },
+            {
+              "sealedAt": "2021-03-03T22:40:39Z",
+              "durationSeconds": "9000"
+            },
+            {
+              "sealedAt": "2021-02-27T15:57:58Z",
+              "durationSeconds": "9000"
+            },
+            {
+              "sealedAt": "2021-02-25T08:57:14Z",
+              "durationSeconds": "7200"
+            },
+            {
+              "sealedAt": "2021-02-23T14:44:18Z",
+              "durationSeconds": "9000"
+            }
+          ]
+        },
+        "retrievals": {
+          "total": "0",
+          "last": null,
+          "failures": "0",
+          "lastFailure": null,
+          "tailTransfers": []
+        }
+      }
+    },
+    "dealsSummary": {
+      "total": "120",
+      "last": "2021-03-17T14:56:51.360Z",
+      "failures": "57",
+      "lastFailure": "2021-03-17T19:41:28.922Z"
+    },
+    "retrievalsSummary": {
+      "total": "0",
+      "last": null,
+      "failures": "0",
+      "lastFailure": null
+    },
+    "updatedAt": "2021-03-17T20:03:02.813Z"
+  },
+  "updatedAt": "2021-03-17T20:03:02.813Z"
+}
+```
+
+</div>
+
+### Make a deal with one of the results
+
+#### Buckets
+
+If you are using [bucket archives](../buckets/archiving) to start data, you can replicate your data with two miners as follows.
+
+```sh
+hub buck archive set-default-config --fast-retrieval --trusted-miners {id1, id2}
+```
+
+Or, if you have your address verified on the network ([learn more here](https://plus.fil.org)), you can make a verified deal. 
+
+```sh
+hub buck archive set-default-config --fast-retrieval --verified-deal --trusted-miners {id1, id2}
+```
```

Comment:
Sounds good. Not sure worth mentioning that now `hub buck archive` will auto-check for bad or misconfigured things.
So in theory the user doesn't necessarily need to change the default config. (but might be a good idea to avoid dense prompts repeating over).